src/averageAges.js

```diff
@@ -34,8 +38,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
-}
+  const women = people.filter(person => !withChildren
+    ? person.sex === 'f'
+    : person.sex === 'f'
+      && people.some(child => child.mother === person.name)
+  );
+
+  const average = women.reduce((acc, woman) => acc
+    + (woman.died - woman.born), 0);
```

Comment:
```suggestion
  const average = women.reduce((acc, woman) =>
    acc + (woman.died - woman.born), 0);
```

Try not to translate arithmetical operators into new lines

Reply:
Aha! Okay, great point!